react/components/SKUSelector/components/Variation.tsx

```diff
@@ -54,13 +63,24 @@ const Variation = ({
       }--${slug(name)} flex flex-column mb7`}
     >
       <div className={`${styles.skuSelectorNameContainer} ma1`}>
-        <span
-          className={`${
-            styles.skuSelectorName
-          } c-muted-2 db t-small overflow-hidden mb3`}
-        >
-          {name}
-        </span>
+        <div className={`${styles.skuSelectorTextContainer} db mb3`}>
+          <span
+            className={`${
+              styles.skuSelectorName
+            } c-muted-1 t-small overflow-hidden`}
+          >
+            {name}
```

Comment:
```suggestion
            {name}{shouldDisplayImageValue && ':'}
```

and `shouldDisplayImageValue` should be `displayImage && selectedItem && showValueNameForImageVariation`, and used here and down below

Reply:
(so to be clear, the result would be `Color: green` instead of `Color green`)

Reply:
...so you can use a darker color and larger font on the color name to improve legibility

Reply:
WWgold does not have any separator but since you and breno think its better, I will add hehe.